style.css

```diff
@@ -0,0 +1,143 @@
+* {
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+li {
+  list-style: none;
+  height: 20px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  text-align: center;
+}
+
+ul {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-wrap: wrap;
+  flex-wrap: wrap;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  margin: 0;
+  padding: 0;
+  padding-top: 12px;
+}
+
+a {
+  text-decoration: none;
+}
+
+.myLogo {
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
+  padding-left: 24px;
+  line-height: 20px;
+  text-align: center;
+}
+
+nav {
+  background-color: #fff;
+}
+
+.toggler-icon {
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  padding-right: 36px;
+}
+
+.font-style {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+}
```

Comment:
- [ ] Great job adding the fonts required in the original design 💯, but you forgot to add a reference to the source of the fonts in the `<head>` of the `.html` file. I suggest you use [Google Fonts.](https://fonts.google.com/)
Example:
```
<head>
   <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100&display=swap" rel="stylesheet">
</head>

```